WebRoot/js/zimbraAdmin/servers/view/ZaServerXFormView.js

```diff
@@ -669,7 +669,7 @@ ZaServerXFormView.BIND_IP_TAB_RIGHTS = [];
 ZaServerXFormView.myXFormModifier = function(xFormObject, entry) {	
 	var headerList = new Array();
 	headerList[0] = new ZaListHeaderItem(ZaServer.A_VolumeName, ZaMsg.VM_VolumeName, null, "100px", false, null, false, true);
-	headerList[1] = new ZaListHeaderItem(ZaServer.A_VolumeRootPath, ZaMsg.VM_VolumeRootPath, null,"200px", false, null, false, true);
+	headerList[1] = new ZaListHeaderItem(ZaServer.A_VolumeRootPath, ZaServer.LBL_VolumeRootPath, null,"200px", false, null, false, true);
```

Comment:
can't we directly use ZaMsg.VM_VolumeRootPath ?
why do we assign ui strings to a variable

Reply:
I think it will be overwritten in a zimlet.

Reply:
@silentsakky indeed, as @k-kato mentioned, this value is overwritten in the zimlet.

Reply:
ok thanks for clarification